### ISSUE TITLE
Test Ansible playbooks using Assert

### ISSUE THREAD

**Issue by dmccuk** [ansible-community/ansible-london-meetup#74]
## Talk title
How to prove Ansible did what it said it did!

## Talk Description
 When we run Ansible to manage server configurations, we assume (if there is no red errors) that it worked. But if you're developing ansible modules for your systems, and want to take the DevOps approach and work on CICD pipelines, you need to have some tests in there to prove that what you asked ansible to do has actually worked.

One of the best ways to do this within ansible, is to use the Assert module. It asserts that a given expression is true. When you combine this with the output of a command that’s registered to a variable, there is almost no limit to what you can test.

## Estimated Duration
15 mins - no more than 20.

## Twitter Handle

LinkedIn? https://www.linkedin.com/in/dennismccarthyuk/

## Sponsorship

I'm just a guy who likes doing stuff with Ansible

**Comment by gundalow** [comment]
@dmccuk This would be a great topic, thank you.
Do you think we could fit in a virtual meetup before AnsibleFest (ie early September)?

**Comment by dmccuk** [comment]
Yes I think there's time. I was planning to run my own meetup in the second
week of September

But I'd be up for waiting and doing an in person one as well if that was a
possibility in October?

**Comment by dmccuk** [comment]
@gundalow How's the planning going for the next meetup. I'm not sure if you saw my previous reply but I'm ready when you are to give a presentation on testing Ansible playbooks using Assert.

**Comment by gundalow** [comment]
Hi,
The Ansible London Meetup is back in-person. I know you made this kind offer to present many years ago, though the [next meetup is Thursday 21st November](https://www.meetup.com/ansible-london/events/303935960/), would you be interested in giving this talk then?
